### PR TITLE
✨Source Delighted: Make Connector Compatible with Builder

### PR DESCRIPTION
**Pull request by btkcodedev**
## What
* Make the connector compatible with the builder

## How
* Import schema to manifest
* Update docs, 
* Completely made compatible with the builder

## User Impact
* No impact, chore for maintainability.

## Can this PR be safely reverted and rolled back?
- [x] YES 💚

### CODE REVIEW

**Comment by btkcodedev** [comment]
Removed $parameters macro, :white_check_mark: 
Embed Spec :white_check_mark: 

GitHub CI Green, ready to merge :white_check_mark: 
image

File: docs/integrations/sources/delighted.md [hunk 1]
```diff
@@ -51,6 +51,7 @@ This source is capable of syncing the following core streams:
 
 | Version | Date       | Pull Request                                             | Subject                                                                                              |
 | :------ | :--------- | :------------------------------------------------------- | :--------------------------------------------------------------------------------------------------- |
+| 0.2.8   | 2024-05-30 | [38142](https://github.com/airbytehq/airbyte/pull/38142) | Make compatable with the builder                                                                     |
```
Comment (girarda) [inline docs/integrations/sources/delighted.md:54]:
```suggestion
| 0.2.8   | 2024-05-14 | [38142](https://github.com/airbytehq/airbyte/pull/38142) | Make compatible with the builder                                                                     |
```

**Review by girarda — APPROVED** [review]
approved pending updating the changelog

**Comment by btkcodedev** [comment]
Change done

**Pull request merged**